duckduckgo/iOS
# show is major tracker network status on the score card

Reviewer: Caine
Asana: https://app.asana.com/0/414235014887631/487711692211751
CC:

**Description**:

Adds an entry to the score card to show if a site is a major tracker network or not (e.g. to clarify why sites like Google get a D grade)

**Steps to test this PR**:
1. Visit Google, view score card, note Site is Major Tracker Network text and red cross
1. Visit DuckDuckGo, view score card, note Site is not Major Tracker Network text and green check
